src/routes/docs/quick-starts/web/+page.markdoc

```diff
@@ -0,0 +1,151 @@
+---
+layout: article
+title: Start with Web
+description: Build JavaScript or Typescript web apps with Appwrite and learn how to use our powerful backend to add authentication, user management, file storage, and more.
+difficulty: beginner
+readtime: 3
+back: /docs/quick-starts
+---
+
+Learn how to add Appwrite to your JavaScript or TypeScript web apps.
+{% section #step-1 step=1 title="Create project" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console).
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+If this is your first time using Appwrite, create an account and create your first project.
+
+Then, under **Add a platform**, add a **Web app**. 
+The **Hostname** should be `localhost` or the domain on which you're hosting your web app. 
+
+{% only_dark %}
+![Add a platform](/images/docs/quick-starts/dark/add-platform.png)
+{% /only_dark %}
+{% only_light %}
+![Add a platform](/images/docs/quick-starts/add-platform.png)
+{% /only_light %}
+
+You can skip optional steps.
+
+{% /section %}
+{% section #step-2 step=2 title="Create project" %}
+You can add the Appwrite Web SDK using CDN with a script tag.
```

Comment:
Shouldn't we encourage people to download the web SDK? 

I feel like the CDN option should be secondary, so maybe reposition this below the NPM option?